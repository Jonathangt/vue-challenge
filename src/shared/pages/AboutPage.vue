<script lang="ts">

import { defineComponent } from 'vue';

export default defineComponent({
	name: 'AboutPage',
	setup() {
		return {}
	},
})
</script>

<template>
	<div>
		About Page
	</div>
</template>

<style scoped></style>
