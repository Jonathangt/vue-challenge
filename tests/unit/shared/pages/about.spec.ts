import { describe, it, expect } from 'vitest'

import { mount } from '@vue/test-utils'
import AboutPage from '../../../../src/shared/pages/AboutPage.vue'

describe('AboutPage', () => {
    it('Matches snapshot', () => {
        const wrapper = mount(AboutPage, {})
        expect(wrapper.html()).toMatchSnapshot()
    })
})
