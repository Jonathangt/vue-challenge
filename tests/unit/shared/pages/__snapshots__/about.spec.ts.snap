// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AboutPage > Matches snapshot 1`] = `"<div> About Page </div>"`;
